js/index.js

```diff
@@ -0,0 +1,58 @@
+
+// - Call ```$('...').val()``` on inputs to get the string value of your user's input
+// - Store user input in ```var city```
+// - Use ```$('...').submit``` to figure out when the user clicks the "submit" button
+// - Create ```if / else if / else``` conditionals to control the flow of your application
+// - Write at least six different lines of pseudocode and display them inline as JavaScript comments
+// - Prevent a form submission using the ```event.preventDefault()``` function
+// - Use the ```$(document).ready()``` handler to delay your code from executing until all DOM assets have been loaded
+// - Use the ```||``` operator in your conditionals to allow for multiple string values to execute ```if/else if``` statement code
+
+
+$(document).ready(function() {
+
+	
+	$('form').submit(changeBackground); 
+
+	function changeBackground(event) {
+		event.preventDefault();
+
+		var cityName = $('#city-type').val();
+		console.log(cityName);
+
+		if (cityName === 'new york' || cityName === 'ny' || cityName === 'nyc') {
```

Comment:
What if I type in `New York` with capital letters? You should add a method when you store the input value so that the user's use of capital letters doesn't affect it.